tests/Behat/WebtoolsConfigContext.php

```diff
@@ -26,4 +26,19 @@ public function webtoolsAnalyicsConfigIsSet(string $id, string $sitepath): void
     $this->setConfig('oe_webtools_analytics.settings', 'sitePath', $sitepath);
   }
 
+  /**
+   * Backup configs that need to be reverted in AfterScenario.
+   *
+   * @BeforeScenario @BackupLacoConfigs
```

Comment:
Probably I would have put this in a step rather than a tag.

Reply:
I think every step should have a business meaning. This is not something a user performs.
That's why I chose to have it as an BeforeScenario method.